flows/vision-zero/Dockerfile

```diff
@@ -15,6 +15,7 @@ COPY ./prefect_bootstrap/id_rsa.pub /root/.ssh/id_rsa.pub
 COPY ./prefect_bootstrap/prefect.config.toml /root/.prefect/config.toml
 RUN chmod 600 /root/.ssh/id_rsa
 RUN chmod 644 /root/.ssh/id_rsa.pub
+RUN cp /root/.ssh/id_rsa.pub /root/.ssh/authorized_keys
```

Comment:
Let the SSH identity of the root user on the container login into localhost via `ssh`